content/en/news/releases/1.19.x/announcing-1.19/upgrade-notes/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Istio 1.19 Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.19.0.
```

Comment:
This would really apply to any 1.19 release. I think we can just go with 1.19 meaning all releases similar to the announce page.
```suggestion
description: Important changes to consider when upgrading to Istio 1.19.
```